src/ray/constants.h

```diff
@@ -0,0 +1,12 @@
+#ifndef RAY_CONSTANTS_H_
+#define RAY_CONSTANTS_H_
+
+/// Length of Ray IDs in bytes.
+constexpr int64_t kUniqueIDSize = 20;
+
+/// Prefix for the object table keys in redis.
+constexpr char ObjectTablePrefix[] = "ObjectTable";
```

Comment:
We should be consistent with naming?: `kObjectTablePrefix`?  

Less importantly, here's another way of initializing string constants:
```
constexpr char* const kObjectTablePrefix = "ObjectTable";
```

Reply:
did the first, for the second, the compiler complains with
```error: ISO C++11 does not allow conversion from string literal to 'char *const'```